cslabs-backend/Controllers/UserModulesController.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using CSLabsBackend.Models;
+
+namespace CSLabsBackend.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
```

Comment:
This controller should be protected. Add the `[Authorize]` attribute to this controller.